sdk/digitaltwins/Azure.DigitalTwins.Core/src/autorest.md

```diff
@@ -7,9 +7,9 @@ Run `generate.ps1` in this directory or run `dotnet build /t:GenerateCode` to ge
 > see <https://aka.ms/autorest>
 
 ``` yaml
-tag: package-2020-10-31
+tag: package-2021-06-30-preview
```

Comment:
How do we ensure we don't ship a "GA" version of this package that is consuming the -preview REST models and defaulting to the -preview API version?

Reply:
Discussed offline - we will make this a preview package.